src/functions/util/editButton.js

```diff
@@ -0,0 +1,44 @@
+const { ButtonBuilder, ActionRowBuilder } = require('discord.js');
+
+module.exports = async (d) => {
+  const data = d.util.aoiFunc(d);
+  const inside = d.unpack();
+  if (data.err) return d.error(data.err);
+
+  let [index, customId, custom, label, style, disabled, emoji, messageId, channelId = d.channel.id] = data.inside.splits;
+
+  const channel = await d.util.getChannel(d, channelId);
+  if (!channel) return d.aoiError.fnError(d, "channel", { inside: data.inside });
+  const message = await d.util.getMessage(channel, messageId);
+  if (!message) return d.aoiError.fnError(d, "message", { inside: data.inside });
+  if (!customId) return d.aoiError.fnError(d, "custom", { inside: data.inside }, "Empty customId was provided");
+  if (isNaN(index) || Number(index) < 1) d.aoiError.fnError(d, "custom", {inside}, "Invalid Index Provided In");
```

Comment:
Error message would return "AoiError: Invalid Invalid Style Provided In [..]", I believe this should be fixed.